File: MakeLifeBetter/composeApp/src/commonMain/kotlin/com/carlosnicolaugalves/makelifebetter/navigation/NavigationItem.kt
package com.carlosnicolaugalves.makelifebetter.navigation

import androidx.compose.material.icons.Icons
import androidx.compose.material.icons.automirrored.filled.Chat
import androidx.compose.material.icons.filled.Event
import androidx.compose.material.icons.filled.Map
import androidx.compose.material.icons.filled.MoreVert
import androidx.compose.material.icons.filled.ShoppingCart
import androidx.compose.ui.graphics.vector.ImageVector

enum class NavigationItem(
    val title: String,
    val icon: ImageVector
) {
    EVENTO("Event", Icons.Filled.Event),
    LOJA("Store", Icons.Filled.ShoppingCart),
    CHAT("Chat", Icons.AutoMirrored.Filled.Chat),
    MAPA("Map", Icons.Filled.Map),
    MORE("More", Icons.Filled.MoreVert)
}

File: MakeLifeBetter/iosApp/iosApp/Navigation/NavigationItem.swift
//
//  NavigationItem.swift
//  iosApp
//
//  Created by Carlos Nicolau Galves on 22/01/26.
//


import SwiftUI

enum NavigationItem: String, CaseIterable {
    case evento = "Event"
    case loja = "Store"
    case chat = "Chat"
    case mapa = "Map"
    case more = "More"

    var title: String {
        return self.rawValue
    }

    var icon: Image {
        switch self {
        case .evento:
            return Image(systemName: "calendar")
        case .loja:
            return Image(systemName: "cart")
        case .chat:
            return Image(systemName: "message")
        case .mapa:
            return Image(systemName: "map")
        case .more:
            return Image(systemName: "ellipsis")
        }
    }
}
